include/natnet_ros_cpp/internal.h

```diff
@@ -68,10 +68,10 @@ class Internal
     void PubRigidbodyMarker(sMarker &data, Internal &internal);
 
     // Publish Single marker as the Rigidbody and TF
-    void PubMarkerPose(sMarker &data, Internal &internal);
+    void PubMarkerPoseWithTrack(sMarker &data, Internal &internal);
```

Comment:
I prefer not change the name of this function as I have already developed on top of this.

Reply:
felt it was more descriptive this way :)